src/lti/Cookie.php

```diff
@@ -2,6 +2,13 @@
 namespace IMSGlobal\LTI;
 
 class Cookie {

```

Comment:
Should we interface this one too, like we have with `Cache`, so we can replace the `set_cookie` behaviour for 5.5 and have the option of using 7.3 syntax when we get there? Does that buy us anything?

Reply:
As far as I can tell, we don't _have_ to change anything when we get to 7.3. I'm reluctant to mess with this much because it feels like there is no reason to change anything (except how we call setcookie internally) when it's available.